cmd/grype/cli/options/grype.go

```diff
@@ -103,6 +104,11 @@ func (o *Grype) AddFlags(flags clio.FlagSet) {
 		"ignore matches for vulnerabilities that are fixed",
 	)
 
+	flags.StringArrayVarP(&o.IgnoreStates,
```

Comment:
It would be nice if it could be invoked with a comma-separate list, like `--ignore-states not-fixed,wont-fix`, but right now I seem to have to pass `--ignore-states not-fixed --ignore-states wont-fix`. `--catalogers` is handled this way in Syft, code at https://github.com/anchore/syft/blob/f6c805797715b3befbf65c291f466f6fbd94fc0b/cmd/syft/cli/options/catalog.go#L100-L104.

Reply:
I've addressed this on the latest commit by moving this parameter to a string and adding a helper for splitting strings on commas and filtering empty entries. I've also added a test for the helper. With these changes in place, the parameter now processes a comma-separated list of states.